specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -221,119 +218,119 @@ JDK™ 1.1 APIs is outside the scope of this specification.
+Jakarta Server Pages, and Jakarta Server Faces interfaces and by the web services support that is a part
+of the Jakarta EE platform.
 
 === HTTPS
 
 Use of the HTTP protocol over the SSL protocol
 is supported by the same client and server APIs as HTTP.
 
-=== Java™ Transaction API (JTA)
+=== Jakarta™ Transactions
 
-The Java Transaction API consists of two parts:
+The Jakarta Transactions consists of two parts:
 
 * An application-level demarcation interface
 that is used by the container and application components to demarcate
 transaction boundaries.
 * An interface between the transaction manager
-and a resource manager used at the Java EE SPI level.
+and a resource manager used at the Jakarta EE SPI level.
 
 === RMI-IIOP (Proposed Optional)
 
 The RMI-IIOP subsystem is composed of APIs that
 allow for the use of RMI-style programming that is independent of the
 underlying protocol, as well as an implementation of those APIs that
 supports both the Java SE native RMI protocol (JRMP) and the CORBA IIOP
-protocol. Java EE applications can use RMI-IIOP, with IIOP protocol
+protocol. Jakarta EE applications can use RMI-IIOP, with IIOP protocol
 support, to access CORBA services that are compatible with the RMI
 programming restrictions (see the RMI-IIOP specification for details).
 Such CORBA services would typically be defined by components that live
-outside of a Java EE product, usually in a legacy system. Only Java EE
+outside of a Jakarta EE product, usually in a legacy system. Only Jakarta EE
 application clients are required to be able to define their own CORBA
 services directly, using the RMI-IIOP APIs. Typically such CORBA objects
 would be used for callbacks when accessing other CORBA objects.
 
-Java EE products must be capable of exporting
-Enterprise JavaBeans components using the IIOP protocol and accessing
-enterprise beans using the IIOP protocol, as specified in the EJB
+Jakarta EE products must be capable of exporting
+Jakarta Enterprise Beans components using the IIOP protocol and accessing
+enterprise beans using the IIOP protocol, as specified in the Jakarta Enterprise Beans
 specification. The ability to use the IIOP protocol is required to
-enable interoperability between Java EE products, however a Java EE
+enable interoperability between Jakarta EE products, however a Jakarta EE
 product may also use other protocols. Requirements for use of the
-RMI-IIOP APIs when accessing Enterprise JavaBeans components have been
-relaxed as of EJB 3.0. See the Enterprise JavaBeans specification for
+RMI-IIOP APIs when accessing Jakarta Enterprise Beans components have been
+relaxed as of EJB 3.0. See the Jakarta Enterprise Beans specification for
 details.
 
 Support for CORBA, including use of IIOP and
-Java IDL, is Proposed Optional as of Java EE 8. See
-link:#a2331[See Pruned Java
+Java IDL, is Proposed Optional as of Jakarta EE 8. See
+link:#a2331[See Pruned Jakarta
```

Comment:
I replaced Java with Jakarta here as it made more sense to me. This needs to be consistent with the rest of the document.